trackmaniaView.js

```diff
@@ -0,0 +1,316 @@
+import { MessageComponentTypes, ButtonStyleTypes } from 'discord-interactions';
+import { getLogger, logProfile } from './log.js';
+import { convertMillisecondsToFormattedTime as convertMS, convertNumberToBase, getDate } from './utils.js';
+
+const log = getLogger();
+
+const MAX_LEADERBOARD_GET = 1000;
+
+export class TrackmaniaView {
```

Comment:
I think the class isn't necessary given that (1) all of the methods here are essentially static (no member variables), and (2) these are in their own file now. Up to you tho